pkg/utils/nodeclass/suite_test.go

```diff
@@ -279,6 +347,10 @@ var _ = Describe("NodeClassUtils", func() {
 				ID:   "ami-custom-id",
 				Tags: map[string]string{},
 			},
+			v1beta1.AMISelectorTerm{
+				ID:   "ami-test",
```

Comment:
You're not adding this in the beginning, and also the previous line expects this to have len = 3, so I don't think this  test should ever succeed.

Reply:
Should be 6, mistakenly, left from testing